deploy/1_tokenized_strategy.js

```diff
@@ -0,0 +1,16 @@
+module.exports = async ({ getNamedAccounts, deployments }) => {
+    const { deploy } = deployments;
+    const { deployer } = await getNamedAccounts();
+
+    console.log("WARN: Ensure TokenizedStrategy has FACTORY address as constant!!!");
+    // console.log("Sleeping for 60 seconds to give a thought...");
+    // await new Promise(r => setTimeout(r, 60000));
```

Comment:
Either remove or uncomment. But do not keep commented code for production branch